### DISCUSSION TITLE
Video not showing up in the actual website, but it shows up in my vscode

### DISCUSSION THREAD

**Discussion by Spurs1012** [community/community#83098] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I successfully used Git LFS to commit and sync my large video files to GitHub. Everything worked fine initially, and the videos appeared when I accessed the website through Visual Studio Code. However, I've noticed that the videos don't show up when I visit the actual website via GitHub. This inconsistency is quite puzzling. Could anyone please provide some assistance and clarification on this issue?

**Comment by lhaibadesigns** [comment]
When using Git LFS (Large File Storage) to handle large files in a Git repository, the behavior you're describing might be related to the way GitHub handles Git LFS objects and the specifics of the website view.

**Reply by Spurs1012** [reply]
Thank you very much! Are there any ways to fix the issue?

**Comment by lhaibadesigns** [comment, marked as answer]
Make sure your Git LFS client is up to date:
```shell
git lfs update
``` 
and then confirm that Git LFS is installed on your local machine, and the Git LFS filters are configured correctly:
```shell
git lfs version
git lfs install
``` 
and re-push LFS objects:
```shell
git lfs push origin "your branch name"
``` 
without "" for sure.
